frontend/src/libraries/explorer-wamp/transactions.ts

```diff
@@ -105,6 +109,29 @@ export interface ReceiptOutcome {
 export interface ReceiptsOutcomeWrapper {
   receiptsOutcome?: ReceiptOutcome[];
 }
+export interface ReceiptOutcomeInfo {
+  predecessor_id: string;
+  receiver_id: string;
+  receipt_id: string;
+  receipt?: ReceiptOutcomeLocalAction;
+}
+
+export interface ReceiptLocalOutcome {
```

Comment:
I don't think we need to have a separate "local" entity. We should be able to construct a local receipt from a transaction and transaction outcome (the receipt ID goes from there).